shipyard.md

```diff
@@ -11,27 +11,42 @@ A shipyard file can consist of any number of stages. A stage has the following p
 
 * **Name**. The name of the stage. This name will be used for the GitOps branch and the Kubernetes namespace to which services at this stage will be deployed to. Future versions will allow to explicitly define the target namespace.
 
-* **Deployment Strategy**. The deployment strategy used to deploy a new version of a service. Keptn supports deployment strategies of type: 
+* **Deployment**. The deployment property is used to specify the deployment strategies. Currently, the following strategies are supported: 
   * `direct`: Deploys a new version of a service by replacing the old version of the service.
   * `blue_green_service`: Deploys a new version of a service next to the old one. After a successful validation of this new version, it replaces the old one and is marked as stable (i.e. it becomes the `primary`-version).
 
-   Future versions of Keptn will also support canary deployments.
+   Future versions of Keptn will also support canary releases.
 
-* **Test Strategy**. Defines the test strategy used to validate a deployment. Failed tests result in an automatic roll-back of the latest deployment in case of a blue/green deployment strategy. Keptn supports tests of type:
+* **Tests**. Currently, in `tests` a single strategy can be used for validating a deployment.
+Failed tests result in an automatic roll-back of the latest deployment in case of a blue/green deployment strategy. Keptn supports tests of type:
   * `functional` 
   * `performance` 
 
+  In future versions of Keptn, it is not necessary to distinguish between functional and performance tests. But it is necessary to define multiple tests for a stage that can either be executed sequentially or in parallel. Therefore, we need a link between the shipyard and the uniform (i.e. using selectors and labels as known by k8s).
+
+* **Remediation**. The remediation property specifies whether remediation actions should automatically be executed in the respective stage. If Keptn receives a problem event (of type `sh.keptn.event.problem.open`), a remediation action is defined for this problem (in the `remediation.yaml` file), and the remediation is set to `automated`, then the remediation action is executed. 
+Keptn supports remediations of type:
+  * `automated`
+  
+* **Approval**. The approval property specifies the kind of approval before promoting an artifact into the next stage. Keptn will/should support the following approval strategies:
+  * `automatic_pass`: The artifact is promoted in an automatic fashion if the evaluation result is `pass`.
+  * `automatic_waring`: The artifact is promoted in an automatic fashion if the evaluation result is `waring` or `pass`.
```

Comment:
typo "waring" vs "warning". in both this line and the following

Reply:
Thank you! I corrected the typos.